org.opent2t.sample.thermostat.superpopular/org.opent2t.sample.thermostat.superpopular.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.opent2t.sample.thermostat.superpopular">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="ambientTemperature" type="d" access="read">
+     <description>ReadOnly, Current temperature measured by the thermostat.</description>
+    </property>
+
+    <property name="targetTemperature" type="d" access="read">
+     <description>Target temperature.</description>
+    </property>
+
+    <property name="targetTemperatureHigh" type="d" access="readwrite">
+      <description>Maximum target temperature (in Celsius).</description>
+    </property>
+
+    <property name="targetTemperatureLow" type="d" access="readwrite">
+      <description>Minimum target temperature (in Celsius).</description>
+    </property>
+    
```

Comment:
I agree with using Celsius units here, as long as the type is decimal not integer so there is no loss of precision when converting on either side.

Note the OCF thermostat schema that we've been discussing actually specifies the units in the property JSON.

(I am an American, and I think non-metric units are silly and old-fashioned.)